src/e

```diff
@@ -90,6 +92,32 @@ program
     'Loads required versions of Xcode and the macOS SDK and symlinks them.  This may require sudo',
   );
 
+program
+  .command('depot-tools')
+  .alias('d')
+  .description('Run a command from the depot-tools directory with the correct configuration')
+  .allowUnknownOption()
+  .action(() => {
+    const args = process.argv.slice(3);
+    let cwd;
+    if (args[0] === 'goma_ctl' || args[1] === 'goma_auth') {
```

Comment:
* should test to confirm args.length > 0

* more importantly, I _think_ you meant that `args[1]` to be `args[0]`?